.github/workflows/r.yml

```diff
@@ -42,6 +43,8 @@ jobs:
         if: always()
         shell: Rscript {0}
         run: |
+          remotes::install_cran("devtools")
+          devtools::load_all()
```

Comment:
Those 2 lines break the `lintr` check (). Running `lintr::lint_package()` will throw errors. Please remove (check e.g. [this](https://github.com/Appsilon/shiny.i18n/blob/main/.github/workflows/main.yml#L43) for reference).

Reply:
This was added as some functions seemed to be not loaded despite being present. [Errors here](https://github.com/Appsilon/shiny.gosling/actions/runs/3748765116/jobs/6366478083)

Reply:
Please change this to follow the simpler pattern that is used in our other packages:
https://github.com/Appsilon/shiny.i18n/blob/main/.github/workflows/main.yml#L35